### ISSUE TITLE
CovPass: [a11y] The QR code image on the main screen is not accessible to VoiceOver.

### ISSUE THREAD

**Issue by MarcoZehe** [Digitaler-Impfnachweis/covpass-ios#39]
The image that shows the QR code on the main screen once a valid vaccination certificate is present, is not accessible. There simply is a big blank space on the screen where VoiceOver doesn't recognize any accessible object. It is difficult for a blind person to know where to hold the iPhone in front of a scanner if we don't know where on our screen the QR code is located. This should be an accessible object, of trait image, and get a sensible accessible label.

**Comment by jleufgen** [comment]
Hi, thanks for Feedback! This will be fixed with one of the next releases. We will improve the app so that it can be read properly with a VoiceOver. Best regards!
